### PR TITLE
Documentation about Pootle FS

### PR DESCRIPTION
**Pull request by unho**
Documentation about Pootle FS

### CODE REVIEW

File: docs/server/commands.rst [hunk 1]
```diff
@@ -893,6 +893,282 @@ and the process will start a watchdog to track any client-side scripts for
...
+
+Pootle_fs
+---------
+
+
+
+
+
+Path options
+------------
```
Comment (dwaynebailey) [inline docs/server/commands.rst]: These headings are wrong, they're at the same level as the title

File: docs/server/commands.rst [hunk 2]
```diff
@@ -893,6 +893,282 @@ and the process will start a watchdog to track any client-side scripts for
 changes. Use this only when developing Pootle.
 
 
+.. _commands#pootle-fs:
+
+Pootle_fs
+---------
```
Comment (dwaynebailey) [inline docs/server/commands.rst]: Pootle FS

File: docs/server/commands.rst [hunk 3]
```diff
@@ -893,6 +893,282 @@ and the process will start a watchdog to track any client-side scripts for
...
+
+
+
+
+
+Path options
+------------
+
+.. django-admin-option:: --pootle_path -P
```
Comment (dwaynebailey) [inline docs/server/commands.rst]: I fixed this previously didn't I?  Short then long, just like every single man page and --help page

**Comment by unho** [comment]
Addressed all the raised concerns.

**Pull request merged**